tests/python/integration/requirements.txt

```
@@ -0,0 +1,3 @@
+pytest==6.2.1
+dataiku-api-client
+git+git://github.com/dataiku/dataiku-plugin-tests-utils.git@master#egg=dataiku-plugin-tests-utils
```

cf, https://github.com/dataiku/dss-plugin-template/pull/9
```suggestion
git+https://github.com/dataiku/dataiku-plugin-tests-utils.git@master#egg=dataiku-plugin-tests-utils
```

---

git+git doesn't work anymore on localhost